src/markd/renderer.cr

```diff
@@ -5,17 +5,18 @@ module Markd
       @last_output = "\n"
     end
 
-    def out(string : String)
-      lit(escape(string))
+    def output(string : String)
+      literal(escape(string))
     end
 
-    def lit(string : String)
+    def literal(string : String)
       @output_io << string
       @last_output = string
     end
 
-    def cr
-      lit("\n") if @last_output != "\n"
+    # render a Line Feed character
+    def write_newline
```

Comment:
`newline` is better than `write_newline`, If this is changed all methods must assigns POS tags to English text, this will a heavy works to be done.